database/migrate/migrations/00015_transaction.sql

```diff
@@ -0,0 +1,46 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE transaction
+(
+    id                  SERIAL       PRIMARY KEY,
+
+    context_id          VARCHAR      NOT NULL, -- batch hash in commit/finalize tx, block hash in update gas oracle tx
+    hash                VARCHAR      NOT NULL,
+    type                SMALLINT     NOT NULL,
+    status              SMALLINT     NOT NULL,
+    rlp_encoding        BYTEA        NOT NULL,
+
+    gas_fee_cap         NUMERIC      NOT NULL,
+    gas_tip_cap         NUMERIC      NOT NULL,
+    gas_price           NUMERIC      NOT NULL,
+    gas_limit           BIGINT       NOT NULL,
+    nonce               BIGINT       NOT NULL,
+    submit_at           BIGINT       NOT NULL,           
```

Comment:
this is confused, in general `xx_at` is a timestamp. So I think can change to `submit_block_number`

Reply:
and reserve the submit_at to record the submit time

Reply:
changed in [88de67b](https://github.com/scroll-tech/scroll/pull/1076/commits/88de67b28bfa9a0183d642c1d6e44a37ffefb732).
and in inserting a new pending transaction, `submit_at` is equal to `created_at`, so we can check the submit time in `created_at`.